packages/editor-ui/src/utils/testData/nodeTypeTestData.ts

```diff
@@ -2647,6 +2647,26 @@ export const nodeTypeTelegramV1 = {
 													default: '',
 													description: 'HTTP or tg:// URL to be opened when button is pressed',
 												},
+												{
+													displayName: 'Web App',
+													name: 'web_app',
```

Comment:
What do you think about changing this to `web_app_url`?

Reply:
Hi, thank you for the review!

The output should contain the following `json` structure:

```json
{
...,
  "reply_markup": {
    "inline_keyboard": [
      [
        {
          "text": "test",
          "web_app": {
            "url": "https://www.google.com/"
          }
        }
      ]
    ]
  },
  ...
}
```
Changing `web_app` to `web_app_url` breaks the proper output structure.

I also followed the structure from the documentation:

https://core.telegram.org/bots/api#inlinekeyboardbutton

where web_app is a [WebAppInfo](https://core.telegram.org/bots/api#webappinfo), which may contain more parameters in the future.